examples/test/test_bert_ptq_cpu_docker.py

```diff
@@ -0,0 +1,36 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+# --------------------------------------------------------------------------
+import os
+import platform
+from pathlib import Path
+
+import pytest
+from utils import check_search_output, patch_config
+
+
+@pytest.fixture(scope="module", autouse=True)
+def setup():
+    """setup any state specific to the execution of the given module."""
+    cur_dir = Path(__file__).resolve().parent.parent
+    example_dir = cur_dir / "bert"
+    os.chdir(example_dir)
+    yield
+    os.chdir(cur_dir)
+
+
+@pytest.mark.parametrize("search_algorithm", ["tpe"])
+@pytest.mark.parametrize("execution_order", ["joint"])
+@pytest.mark.parametrize("system", ["docker_system"])
+@pytest.mark.parametrize("olive_json", ["bert_ptq_cpu.json"])
```

Comment:
If we only test one value for these args, we can remove the parametrized annotation. Same to other tests.

Reply:
Yes, we can definitely clean up the parameterization for sure.